src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -149,6 +149,20 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
   protected String getRecipeList(HttpServletRequest request, HttpServletResponse response) {
     String creatorToken = request.getParameter("token");
 
+    SortingMethod sortingMethod;
+    if (request.getParameter("sort") == null) {
+      sortingMethod = SortingMethod.TOP;
+    } else {
+      sortingMethod = SortingMethod.valueOf(request.getParameter("sort"));
+    }
+
+    int page;
+    if (request.getParameter("page") == null) {
+      page = -1;
```

Comment:
I feel using -1 (and pass this around) to mean "load all receipts" make things less readable. I'd rather overload methods in Firestore.db to have versions with and without page number, with the version without page number to return all recipes.

Reply:
Yup; I didn't want to overload at first because it was a lot of copying and pasting, but since java doesn't have optional method args I guess this is how it usually is in Java.

Reply:
Overloads added in 57e8ad6 and used in db3d476.